build.fsx

```diff
@@ -488,12 +489,19 @@ Target "Protobuf" <| fun _ ->
 // Documentation 
 //--------------------------------------------------------------------------------  
 Target "DocFx" (fun _ ->
+    // build the project with samples
     let docsExamplesSolution = "./docs/examples/DocsExamples.sln"
     DotNetCli.Restore (fun p -> { p with Project = docsExamplesSolution })
     DotNetCli.Build (fun p -> { p with Project = docsExamplesSolution; Configuration = configuration })
 
-    let docsPath = "./docs"
+    // install MSDN references
```

Comment:
Nice! so this will allow some of the refs we use to point back to the MSDN docs from our own site going forward?

Reply:
yes